source/docs/essentials/extend-the-theme.md

```diff
@@ -1,6 +1,129 @@
 ---
+id: extend-the-theme
 title: Extend the theme
 ---
 
-> Content coming soon. See
-> [https://gitlab.com/front-commerce/front-commerce/issues/13](https://gitlab.com/front-commerce/front-commerce/issues/13)
+## Understanding theme overrides
+
+Front-Commerce web application is a fully featured e-commerce universal React application
+aimed at providing a sane base for your own theme.
+Even though you could reuse pages and components individually in a totally different theme,
+most of the time you might find easier and faster to start with the base theme and
+iterate from there.
+
+**Theme override is the mechanism allowing to extend, adapt, rewrite almost everything contained in the base theme.** It is what allows you to:
+- upgrade Front-Commerce and benefit from the latest component improvements,
```

Comment:
I'd add `customize the base theme` as a first item. It's close to the `extension` part but since we don't really have any extensions yet...